File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UpdatePlantBatchesTagRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class UpdatePlantBatchesTagRequest {
    @JsonProperty("Group")
    public String group;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("NewTag")
    public String newTag;
    @JsonProperty("ReplaceDate")
    public String replaceDate;
    @JsonProperty("TagId")
    public Integer tagId;
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/UpdatePlantBatchesTagRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class UpdatePlantBatchesTagRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("Group")]
        public string? Group { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Id")]
        public int? Id { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("NewTag")]
        public string? NewTag { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ReplaceDate")]
        public string? ReplaceDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("TagId")]
        public int? TagId { get; set; }
    }
}
